styles.css

```diff
@@ -121,13 +198,118 @@ p {
   display: flex;
   position: relative;
   flex-direction: column;
-  padding-top: 20%;
+  padding-top: 5%;
+}
+
+.works #web_img {
+  display: none;
 }
 
 .works h2 {
   padding: 20px;
 }
 
+.card_content-web {
+  display: none;
+}
+
+#mobile-title {
+  display: none;
+}
+
+@media screen and (min-width: 800px) {
+  .card_content-web {
+    display: grid;
+    grid-template-columns: 1fr;
+  }
+}
+
+.item1 {
+  display: grid;
+  grid-template-columns: 1.5fr 2.5fr;
+}
+
+.item1-rightpanel {
+  width: 100%;
+  height: 100%;
+  padding-left: 42px;
+}
```

Comment:
Kindly ensure that the project cards properly as well on all desktop sizes (768px and wider).

Your design at 809px:
![image](https://user-images.githubusercontent.com/20554583/206877652-52b64c6b-8a38-4d91-b985-a528bf958cc7.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/206877667-48cf4cbf-659e-488f-948f-c61bb94e94e2.png)